### ISSUE TITLE
For November 2018

### ISSUE THREAD

**Issue by HarryMacfinned** [bisq-network/compensation#172]
## Summary

> _Specify the total amount of BSQ you are requesting, and the BSQ address that amount should be paid to:_

 - BSQ requested: `2400`
 - BSQ address: B19yxYCRTVaDfRfucrH6aEzeZpoVTv8Je8S

This compensation request :
- follows the https://docs.bisq.network/roles.html#compensation recommandations
- applies the 1BSQ=1USD rule
- takes into account the time spent for the roles
- mimicks october compensation request (for the roles)

### Substantive reports

- Support | https://github.com/bisq-network/roles/issues/64#issuecomment-442775533 | (BSQ  requested = 2000)
Evaluation is based and intended in line with : previous compensation requests, https://github.com/bisq-network/compensation/issues/24 , and other support compensation requests.
- Fees and connectivity monitoring | see the weekly reports in https://github.com/bisq-network/roles/issues/10 | (BSQ requested = 200)
TL;DR: there was zero catch this month (nodes are running well).
- Prices monitoring | https://github.com/bisq-network/roles/issues/10#issuecomment-442748989 | (BSQ requested = 200)

## Other points (BSQ requested = 0)
- Clearnet Bisq webpages monitoring -> see the slack #bisq-monitor channel: https://bisq.slack.com/messages/C81CHDYFM .(This monitoring follows a little issue with the https://markets.bisq.network/ page)
- investigated a bit with slack replacement proposal https://github.com/bisq-network/proposals/issues/54

## Contributions in progress
- participation to the v0.9.0 tests = WIP => I may or will ask for compensation later on
- I'm organizing for realtime monitoring/reporting. Should be on an onion at first.
- I'm investigating for BTC nodes monitoring/reporting.

**Comment by ripcurlx** [comment]
Closing as complete, see https://github.com/bisq-network/compensation/issues/165#issuecomment-444496356.
